### PR TITLE
Update footer and strip background colors

### LINKED ISSUE #17: Update juju.is footer to match jaas.ai
Please can we update the footer to use the one currently on [jaas.ai](https://jaas.ai/). We may need to change the background colour of the last "Learn more" section, let's see.

This footer will also be used on the Charmhub site to make them all consistent.

### LINKED ISSUE #18: Update the strip background colour
Can we update the `p-strip--light` background colour from #F7F7F7 to #F0F5F7 to match jaas.ai. Looks like it's called `p-strip--accent` on jaas.ai

Thanks.

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done 

- Updated the footer to match jaas.ai
- Updated the strip accent colors to match jaas.ai

Fixes #17, Fixes #18

### CODE REVIEW

**Comment by hatched** [comment]
Where the content meets the footer it's a bit weird having that white wedge there. Is this intentional @spencerbygraves ?

Screen Shot 2020-01-21 at 11 36 35 AM

**Comment by barrymcgee** [comment]
@spencerbygraves I've made the changes to the suru footer. The social icon changes will come for free with the next Vanilla bump to 2.5.0

File: static/sass/styles.scss [hunk 1]
```diff
@@ -86,3 +87,14 @@
     }
   }
 }
+
+// Homepage footer overide
+.p-strip--suru-top.is-homepage::before {
```
Comment (anthonydillon) [inline static/sass/styles.scss]: If only these strips were in Vanilla this would just working using mask.

File: templates/base.html [hunk 2]
```diff
@@ -176,31 +176,54 @@
...
-    <footer class="p-strip">
+    <footer class="p-strip--suru-top has-accent is-dark {% if request.path == '/' %}is-homepage{% endif %}"
+      style="background-size: auto 100%, cover; background-image: url('https://assets.ubuntu.com/v1/8951dc67-suru-left.svg'), linear-gradient(266deg, #044f66, #022935)">
       <div class="row">
-        <div>
-          <p class="u-no-max-width">
-            © {{ current_year }} Canonical Ltd. Ubuntu and Canonical are
-            registered trademarks of Canonical&nbsp;Ltd.
+        <div class="col-8">
+          <p class="u-no-max-width">© 2020 Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical Ltd.
```
Comment (anthonydillon) [inline templates/base.html]: Why not use current year variable?

**Pull request merged**